components/Composition/FAnnouncementCard.jsx

```diff
@@ -0,0 +1,88 @@
+import { StyleSheet, View } from 'react-native';
+import { FImage } from 'components/Composition/FImage';
+import { getHalfBorderRadius } from 'utils/getHalfBorderRadius';
+import colors from 'themes/colors';
+import sizes from 'themes/sizes';
+import opacities from 'themes/opacities';
+import fonts from 'themes/fonts';
+import { FHeading } from 'components/Composition/FHeading';
+import React from 'react';
+import { TouchableWithoutFeedback } from 'react-native-web';
+import { FHeadingWithIcon } from 'components/Composition/FHeadingWithIcon';
+import placements from 'themes/placements';
+import icons from 'themes/icons';
+
+export const FAnnouncementCard = ({ width, data }) => {
+  const cropText = (text, number) => `${text.slice(0, width / number)}...`;
+  return (
+    <TouchableWithoutFeedback>
+      <View style={{
+        width,
+        ...styles.container,
+      }}
+      >
+        <FImage
```

Comment:
Przyjrzyj sie lepiej jakie propsy i jak wyglada komponent FImage. On ma propsy width i height oddzielnie i nie dajemy tego do containerStyle, to sa dodatkowe opcje jakby ktoś chciał dodać paddingi i takie pierdoły. Musisz mu dać też resizeMode i imagePath dać jako null. Nie wrzuciłam jeszcze zmian z proptypes ale tutaj masz mocka jak to bedzie wyglądać wiec do tego trzeba sie dostosować

FImage.propTypes = {
  width: PropTypes.oneOfType([
    PropTypes.string,
    PropTypes.number,
  ]).isRequired,
  height: PropTypes.oneOfType([
    PropTypes.string,
    PropTypes.number,
  ]).isRequired,
  imagePath: PropTypes.oneOfType([
    PropTypes.string,
    PropTypes.number,
  ]).isRequired,
  resizeMode: PropTypes.oneOf(['cover', 'contain']).isRequired,
  networkImageUrl: PropTypes.string,
  imageWidth: PropTypes.oneOfType([
    PropTypes.string,
    PropTypes.number,
  ]),
  imageHeight: PropTypes.oneOfType([
    PropTypes.string,
    PropTypes.number,
  ]),
};